README.rst

```diff
@@ -131,6 +131,7 @@ We use a different list of plugins for Open By Default::
         canada_obd canada_package wet_boew_gcweb scheming_datasets
         fluent cloudstorage
 
+   ckan.extra_resource_fields = language
```

Comment:
I forgot to ask: Can we apply this setting from our plugin so it's not an extra manual step?